openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2285,6 +2294,9 @@ Wallet Providers may also provide a marketplace where Issuers can register to be
    
    -13
 
+   * change the structure of `proof_types` from an array to a `proofs_supported` map that contains a required `proof_signing_alg_values_supported` parameter
+   * renamed `credential_configurations_supported` to `credential_signing_alg_values_supported` to clarify the purpose of the parameter
```

Comment:
```suggestion
   * renamed `cryptographic_suites_supported` to `credential_signing_alg_values_supported` to clarify the purpose of the parameter
```

Reply:
This one is not applied yet. The history text sends wrong?